tests/plots/test_image.py

```diff
@@ -24,10 +25,33 @@ def test_pil(tmp_dir):
     assert (tmp_dir / live.plots_dir / LiveImage.subfolder / "image.png").exists()
 
 
+def test_pil_omitting_extension_doesnt_save_without_valid_format(tmp_dir):
+    live = Live()
+    img = Image.new("RGB", (10, 10), (250, 250, 250))
+    with pytest.raises(InvalidDataTypeError, match="has not supported type"):
```

Comment:
As mentioned above, would be better to have a separate error here for invalid/missing output extension.